projects/libyuv/libyuv_fuzzer.cc

```diff
@@ -0,0 +1,70 @@
+/* # Copyright 2020 Google Inc. */
+/* # */
+/* # Licensed under the Apache License, Version 2.0 (the "License"); */
+/* # you may not use this file except in compliance with the License. */
+/* # You may obtain a copy of the License at */
+/* # */
+/* #      http://www.apache.org/licenses/LICENSE-2.0 */
+/* # */
+/* # Unless required by applicable law or agreed to in writing, software */
+/* # distributed under the License is distributed on an "AS IS" BASIS, */
+/* # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. */
+/* # See the License for the specific language governing permissions and */
+/* # limitations under the License. */
+/* # */
+/* ################################################################################ */
+
+#include <iostream>
+#include <cstdlib>
+#include <ctime>
+#include "libyuv.h"
+using namespace std;
+
+unsigned int fastrand_seed;
+inline int fastrand()
+{
+    fastrand_seed = fastrand_seed * 214013u + 2531011u;
```

Comment:
If i run this fuzzer today and then I run the same binary tomorrow, can the same testcase/input/`data` give different results?
I think this would be a major problem if I understand this is the case. You will get unreproducible crashes and fuzzing won't work properly because different processes will have different ideas on what testcases are doing.
I think https://github.com/google/fuzzing/blob/master/docs/split-inputs.md might allow you to do something similar in a way that is deterministic wrt the input (`data`). Can you use this? Or please let me know if I'm misunderstanding.

Reply:
You are right!  I have understanded this problem, and will read the **split-inputs.md** , fix this problem.